lib/consumer.js

```diff
@@ -112,7 +112,7 @@ Consumer.prototype.init = function () {
             return self.emit('error', err);
         }
 
-        if (self.options.fromOffset)
+        if (!self.options.fromOffset)
```

Comment:
That function should be exited with `return self.fetch();` so it will not fetch commited offset with `self.fetchOffset`. Does it work for you with your fix? I can't force offset in payload regardless of `fromOffset` value. I have not yet tried your fix.